internal/targetedretries/ruby_cucumber_substitution.go

```diff
@@ -12,28 +12,28 @@ import (
 type RubyCucumberSubstitution struct{}
 
 func (s RubyCucumberSubstitution) Example() string {
-	return "bundle exec cucumber {{ examples }}"
+	return "bundle exec cucumber {{ scenarios }}"
```

Comment:
Just making these align while in here. I'll update docs. Again, this has no usage right now